### PR TITLE
Misc screen tweaks and cleanup

### PR DESCRIPTION
**Pull request by cgzones**

### CODE REVIEW

**Comment by BenBE** [comment]
Typo in second commit "ScreenDefaul**t**s" …

File: Settings.c [hunk 1]
```diff
@@ -242,8 +242,10 @@ static void ScreenSettings_readFields(ScreenSettings* ss, Hashtable* columns, co
    char** ids = String_split(trim, ' ', NULL);
    free(trim);
 
-   unsigned int i, j;
-   for (j = 0, i = 0; ids[i]; i++) {
+   /* reset default fields */
+   memset(ss->fields, '\0', LAST_PROCESSFIELD * sizeof(ProcessField));
+
+   for (unsigned int j = 0, i = 0; ids[i]; i++) {
```
Comment (BenBE) [inline Settings.c:248]:
```suggestion
   for (size_t j = 0, i = 0; ids[i]; i++) {
```

File: Process.c [hunk 2]
```diff
@@ -1026,8 +1026,7 @@ const ProcessClass Process_class = {
       .delete = Process_delete,
       .compare = Process_compare
    },
-   .writeField = Process_writeField,
-   .getCommandStr = Process_getCommandStr,
+   .writeField = Process_writeField
```
Comment (BenBE) [inline Process.c:1029]:
```suggestion
   .writeField = Process_writeField,
```

Why not keep this comma?

**Comment by Explorer09** [comment]
33cb12ac5ed21e4afe8bc7b0384fe54e6433e24b
"Mark ScreenDefauls const"
Typo in commit description.

**Comment by Explorer09** [comment]
fd207098f9f9b5eea6f672a34014a26e773976e7
"Set correct default sorting directtion"
Typo. ("direction")

**Review by BenBE — APPROVED** [review]

**Pull request merged**